drivers/slipdev/stdio.c

```diff
@@ -34,9 +34,11 @@ static void _isrpipe_write(void *arg, uint8_t data)
 static void _init(void)
 {
     /* intentionally overwritten in netdev init so we have stdio before
-     * the network device is initialized is initialized */
+     * the network device is initialized */
     uart_init(slipdev_params[0].uart, slipdev_params[0].baudrate,
               _isrpipe_write, &stdin_isrpipe);
+    
```

Comment:
Trailing whitespaces (Github somehow shows that as a complete line removal, but I just want you to remove the non-linebreaking whitespaces)

```suggestion

```